pycypher/src/Makefile.am

```diff
@@ -0,0 +1,49 @@
+generated_bindings = getters.py node_types.c operators.c props.c
+
+.NOTPARALLEL: $(generated_bindings)
```

Comment:
Make is often run using parallelization (by setting `MAKEFLAGS=-j4` for example).

This rule covers four output targets (getters.py, node_types.c, operators.c & props.c). If make was running in parallel, it would invoke this rule concurrently for each output.

In this case, however, your generation script builds all four output files at once. So there is no need to run in parallel as all dependencies will be fulfilled by a single invocation.